Makefile

```diff
@@ -41,8 +41,8 @@ Cargo.lock: Cargo.toml
 	cargo update --workspace
 
 install_rust: Cargo.lock
-	cargo install soroban-cli --version 20.2.0
-	cargo install --path ./cmd/crates/soroban-test/tests/fixtures/hello --root ./target --debug --quiet
+	#cargo install soroban-cli --version 20.2.0
+	#cargo install --path ./cmd/crates/soroban-test/tests/fixtures/hello --root ./target --debug --quiet
```

Comment:
Can you check whether or not this actually affects things now that we've resolved the circular dependencies? If it's still possible to install the `soroban` CLI from Cargo, we can still use it to build the `test_hello_world.wasm` rather than hard-coding it.

Reply:
I specifically mean the first line (L44) not the one involving fixtures.

Reply:
Discussed offline !! [will add it back later if nothing breaks]